go/adbc/adbc.go

```diff
@@ -326,6 +336,7 @@ const (
 // or desire for it.
 type Driver interface {
 	NewDatabase(opts map[string]string) (Database, error)
+	NewDatabaseWithContext(ctx context.Context, opts map[string]string) (Database, error)
```

Comment:
This should be in `ext.go`

Reply:
Moved. Let me know if the wrapping interface name is appropriate.